dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentParamsMapper.java

```diff
@@ -104,8 +105,9 @@ public EnrollmentQueryParams map( RequestParams requestParams )
         validateTrackedEntity( requestParams.getTrackedEntity(), trackedEntity );
 
         User user = currentUserService.getCurrentUser();
-        Set<String> orgUnitIds = parseUids( requestParams.getOrgUnit() );
-        Set<OrganisationUnit> orgUnits = validateOrgUnits( user, orgUnitIds, program );
+        Set<UID> orgUnitUids = validateDeprecatedUidsParameter( "orgUnit", requestParams.getOrgUnit(), "orgUnits",
+            requestParams.getOrgUnits() );
+        Set<OrganisationUnit> orgUnits = validateOrgUnits( user, orgUnitUids, program );
 
         EnrollmentQueryParams params = new EnrollmentQueryParams();
         params.setProgram( program );
```

Comment:
Just wonder if there isn't a builder for the `EnrollmentQueryParams`

Reply:
No there is not, this is a task for another time 😋